modules/logging-loki-zone-fail-recovery.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-loki-zone-fail-recovery_{context}"]
+= Recovering Loki pods from failed zones
+
+In  {product-title} a zone failure happens when specific availability zone resources become inaccessible. Availability zones are isolated areas within a cloud provider's data center, aimed at enhancing redundancy and fault tolerance. If your cluster isn't configured to handle this, a zone failure can lead to service or data loss.
```

Comment:
We should be precise on which cluster we mean with this sentence. I believe you mean OCP cluster here, right?

```
If your cluster isn't configured to handle this, a zone failure can lead to service or data loss.
```